### PR TITLE
merge ffu's 0.8 behaviors branch

### PR DESCRIPTION
**Pull request by morethanreal**
PTAL @cdata

### CODE REVIEW

File: iron-multi-selectable.html [hunk 1]
```diff
@@ -0,0 +1,127 @@
+<!--
```
Comment (notwaldorf) [inline iron-multi-selectable.html]: nit: `license@`?

File: test/multi.html [hunk 2]
```diff
@@ -115,7 +115,7 @@
         assert.equal(deselectEventCounter, 1);
       });
 
-      test('toggle multi from true to false', function() {
+      /* test('toggle multi from true to false', function() {
```
Comment (notwaldorf) [inline test/multi.html]: is this commented out because it's failing? do we know why it's failing? (i'm sorry!)

Comment (morethanreal) [inline test/multi.html]: Don't know. cc @frankiefu

File: iron-multi-selectable.html [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="iron-selectable.html">
+
+<script>
+
+  Polymer.IronMultiSelectableBehavior = [
```
Comment (cdata) [inline iron-multi-selectable.html]: Where is the extends keyword when you need it? :bike:

Comment (cdata) [inline iron-multi-selectable.html]: Nice specialization of `selectable` btw :beer:

File: iron-selectable.html [hunk 4]
```diff
@@ -220,36 +274,25 @@
...
-    toggleSelected: function(value) {
-      var i = this.selected.indexOf(value);
-      if (i < 0) {
-        this.selected.push(value);
-      } else {
-        this.selected.splice(i, 1);
+    _activateHandler: function(e) {
+      var t = e.target;
+      var items = this.items;
+      while (t && t != this) {
```
Comment (cdata) [inline iron-selectable.html]: I say boo to abbreviations :ghost:

File: iron-selectable.html [hunk 5]
```diff
@@ -12,50 +12,110 @@
...
+    },
 
-    /* The attribute to set on elements when selected */
-    /* selectedAttribute: 'active', */
+    observers: [
+      '_updateSelected(attrForSelected, selected)'
+    ],
 
     excludedLocalNames: {
       'template': 1
```
Comment (cdata) [inline iron-selectable.html]: This is kind of a clunky way to expose the exclusion list to an element that uses this behavior and wants to exclude other things. The element author needs to know that the old list contained `template` if they are over-writing it in their element. :bicyclist:

**Comment by cdata** [comment]
I didn't have anything valuable to add, really. Just opinions :) LGTM :+1:

**Comment by notwaldorf** [comment]
i only have opinions about the missing `@license`, otherwise LGTM :trophy:

File: iron-selectable.html (hunk 5 above)
Comment (morethanreal) [inline iron-selectable.html]: I filed #4 to track, but merging the PR to use this for menus since this is not blocking.

**Pull request merged**